cumulusci/tasks/bulkdata/mapping_parser.py

```diff
@@ -0,0 +1,58 @@
+from typing import Dict, List
+from logging import getLogger
+
+from pydantic import Field, validator, ValidationError
+
+from cumulusci.utils.yaml.model_parser import CCIDictModel
+
+LOGGER_NAME = "MAPPING_LOADER"
```

Comment:
```suggestion
logger = getLogger(__name__)
```

and then use `logger` below. To avoid constructing multiple instances, and for consistency in naming with what we do in other modules.